ansible/roles/matrix/files/hooks/user

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import sys
+password = json.loads(os.environ['LIQUID_HOOK_DATA'])['password']
+[username] = sys.argv[1:]
+
+cmd = [
+    "/opt/matrix/synapse/bin/python",
+    "/opt/matrix/synapse/local/bin/register_new_matrix_user",
+    "--config", "/opt/matrix/synapse/homeserver.yaml",
+    "--user", username,
+    "--password", password,
+    "--admin",  # required because of https://github.com/matrix-org/synapse/issues/2310
```

Comment:
Let's create an issue for ourselves to fix this. We can't let everybody be an admin.

Reply:
See https://github.com/liquidinvestigations/liquidinvestigations/issues/85